MAUI/UIKit/page-and-template-dependencies.md

```diff
@@ -0,0 +1,1574 @@
+---
+layout: post
+title:  XAML Templates | Page Template Dependencies of UI Kit | Syncfusion
+description: The list of screen-wise resources and NuGet packages required for each Individual MAUI Essential UI kit Screens.
+platform: maui
+control: UI kit
+documentation: ug
+---
+
+This page list the dependencies ( converters, rendereres and NuGet packages) of each XAML page in Essential UI Kit project  
+
+# Page Template Dependencies of MAUI UI Kit
```

Comment:
# Page Template Dependencies of .NET MAUI UI Kit

Reply:
Changed In Latest Commits